test/monster_test.rb

```diff
@@ -117,6 +117,12 @@
     slice_wav.duration.must_equal 5
   end
 
+  it 'can cut a section from the top of an audio file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.